lib/sse.js

```diff
@@ -0,0 +1,40 @@
+const { status, header } = require('node-res')
+
+class SSE {
+  constructor () {
+    this.subscriptions = new Set()
+    this.counter = 0
+  }
+
+  // Subscribe to a channel and set initial headers
+  subscribe (req, res) {
+    req.socket.setTimeout(0)
```

Comment:
Maybe we can allow a class option to change this value?

Reply:
Well actually for doing SSR we need to set it to `0` or `Infinity`